### PR TITLE
Fix warnings when executing CLI without arguments.

### PR DESCRIPTION
**Pull request by srlowe**
An edge case call to `capitano.execute` was not promisified, which was generating warnings in Node 12.

Note that the test will not catch the problem until we stop sentry masking errors/warnings.  Has been tested with sentry disabled though.

Change-type: patch
Signed-off-by: Scott Lowe <scott@balena.io>
Resolves: #1491

---
##### Contributor checklist

- [ ] Introduces security considerations
- [ ] Affects the development, build or deployment processes of the component

### CODE REVIEW

File: lib/app-capitano.coffee [hunk 1]
```diff
@@ -26,7 +26,8 @@ capitano.permission 'user', (done) ->
 capitano.command
 	signature: '*'
 	action: ->
-		capitano.execute(command: 'help')
+		capitanoExecuteAsync = Promise.promisify(capitano.execute)
+		capitanoExecuteAsync(command: 'help')
```
Comment (pdcastro) [inline lib/app-capitano.coffee]: Nice 👍   What might be, I think, more "idiomatic" of Capitano (i.e. matching the [documentation](https://www.npmjs.com/package/capitano#capitanoexecutecli-callback) and how most CLI commands are currently implemented, e.g. [help.coffee](https://github.com/balena-io/balena-cli/blob/v11.17.1/lib/actions/help.coffee#L146-L150)) is to use a callback:
```js
	action: (params, options, done) ->
		capitano.execute(command: 'help', done)
```
I've tested it with Node 12, it seems to fix the issue just as well.

Comment (srlowe) [reply lib/app-capitano.coffee]: Sure, will modify +1
